### PR TITLE
Closes #182. | Implement dataloader for `roots_vi_ted`

### LINKED ISSUE #182: Create dataset loader for ROOTS_vi_ted
Dataloader name: `roots_vi_ted/roots_vi_ted.py`
DataCatalogue: http://seacrowd.github.io/seacrowd-catalogue/card.html?roots_vi_ted

|     Dataset| roots_vi_ted |
|-------------|---|
| Description | ROOTS_vi_ted is a subset of Vietnamese in ted_talks_iwslt datasets. ted_talks_iwslt is a collection of the original Ted talks and their translated version. The translations are available in more than 109+ languages, though the distribution is not uniform. |
| Subsets | - |
| Languages | vie |
| Tasks | Language Modeling |
| License | Creative Commons Attribution Non Commercial 2.0 (cc-by-nc-2.0) |
| Homepage | https://huggingface.co/datasets/bigscience-data/roots_vi_ted_talks_iwslt |
| HF URL | https://huggingface.co/datasets/bigscience-data/roots_vi_ted_talks_iwslt |
| Paper URL | https://arxiv.org/abs/2303.03915 |

### PR DESCRIPTION
**Pull request by chenxwh**
Closes #182.

### Checkbox
- [x] Confirm that this PR is linked to the dataset issue.
- [x] Create the dataloader script `seacrowd/sea_datasets/my_dataset/my_dataset.py` (please use only lowercase and underscore for dataset naming).
- [x] Provide values for the `_CITATION`, `_DATASETNAME`, `_DESCRIPTION`, `_HOMEPAGE`, `_LICENSE`, `_URLs`, `_SUPPORTED_TASKS`, `_SOURCE_VERSION`, and `_SEACROWD_VERSION` variables.
- [x] Implement `_info()`, `_split_generators()` and `_generate_examples()` in dataloader script.
- [x] Make sure that the `BUILDER_CONFIGS` class attribute is a list with at least one `SEACrowdConfig` for the source schema and one for a seacrowd schema.
- [x] Confirm dataloader script works with `datasets.load_dataset` function.
- [x] Confirm that your dataloader script passes the test suite run with `python -m tests.test_seacrowd seacrowd/sea_datasets/<my_dataset>/<my_dataset>.py`.
- [x] If my dataset is local, I have provided an output of the unit-tests in the PR (please copy paste). This is OPTIONAL for public datasets, as we can test these without access to the data files.

### CODE REVIEW

**Comment by ryanignatius** [comment]
hi @chenxwh thank you for the PR

about the data, since it is publicly accessible after user has accepted the dataset's acknowledgement, could we append "Before using this dataloader, please accept the acknowledgement at https://huggingface.co/datasets/bigscience-data/roots_vi_ted_talks_iwslt and use huggingface-cli login for authentication." to the _DESCRIPTION?

also remove `train-00000-of-00001.parquet` file from data directory and change _LOCAL to false

File: seacrowd/sea_datasets/roots_vi_ted/roots_vi_ted.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+            features = datasets.Features(
+                {
+                    "text": datasets.Value("string"),
+                    "meta": datasets.Value("string"),
+                }
+            )
+
+        elif self.config.schema == "seacrowd_ssp":
+            features = schemas.self_supervised_pretraining.features
+            features["meta"] = {
```
Comment (ryanignatius) [inline seacrowd/sea_datasets/roots_vi_ted/roots_vi_ted.py:92]: I think we can omit this meta here and only use the text for the seacrowd_ssp schema

File: seacrowd/sea_datasets/roots_vi_ted/roots_vi_ted.py [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+_LOCAL = True
+
+_SUPPORTED_TASKS = [Tasks.SELF_SUPERVISED_PRETRAINING]
+
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+class RootsViTed(datasets.GeneratorBasedBuilder):
```
Comment (ryanignatius) [inline seacrowd/sea_datasets/roots_vi_ted/roots_vi_ted.py:55]: please add `Dataset` suffix to class name
```suggestion
class RootsViTedDataset(datasets.GeneratorBasedBuilder):
```

**Comment by chenxwh** [comment]
Thank you! Updated accordingly

**Comment by holylovenia** [comment]
Replacing @gentaiscool with @MJonibek due to no response.

**Review by MJonibek — CHANGES_REQUESTED** [review]
Hello @chenxwh , everything works, just small formating is needed. Please, can you run 

`make check_file=seacrowd/sea_datasets/roots_vi_ted/roots_vi_ted.py`

image

To solve two warnings you need to delete `import os` and delete space in the end of the 35 line.

**Comment by chenxwh** [comment]
thanks @MJonibek just updated

**Review by MJonibek — APPROVED** [review]
@chenxwh Thank you for the fast response, LGTM

**Review by ryanignatius — APPROVED** [review]

**Pull request merged**